backend/package.json

```diff
@@ -4,7 +4,8 @@
   "description": "",
   "main": "server.js",
```

Comment:
Should now be
  "main": "backend/src/server.js",

Reply:
good catch! i think since this is in the backend dir though it should be `src/server.js`. unless you want to rollup the dependencies into one `package.json` at the root of this repo.

updated here: d6c8a394f2118d9d13764d7f4d6561e8505f1fa1